lib/school_house_web/templates/layout/_footer.html.heex

```diff
@@ -1,5 +1,4 @@
 <footer class="mt-16 bg-footer dark:bg-footer-dark" aria-labelledby="footerHeading">
-  <h2 id="footerHeading" class="sr-only">Footer</h2>
```

Comment:
I am not sure I can get rid of this element.

Reply:
I would convert this one also to a `p`. The Tailwind CSS class `sr-only` is for "screen reader only", that is, it will be visually hidden, but available on screen readers.

Reply:
@jaimeiniesta. I address in here. cc6129a